### PR TITLE
Reenable uglifyjs

### PR DESCRIPTION
**Pull request by ph1p**
I tried several options and found out, that the **unused** flag must be set to **false**.

>unused (default: true) -- drop unreferenced functions and variables (simple direct variable assignments do not count as references unless set to "keep_assign")

**Source:** https://github.com/mishoo/UglifyJS2/tree/harmony#parse-options

**Issue:** #410

### CODE REVIEW

**Comment by perry-mitchell** [comment]
Thanks @ph1p - This will obviously need a test at some stage, for the build. The issue was only present in production builds, so I'll let @sallar confirm that this setting functions correctly.

**Comment by sallar** [comment]
Thanks @ph1p. I'll test this and if it's fixed then I'll merge :) Cheers

**Comment by ph1p** [comment]
@sallar I've written a test to check the focus on input fields. It's not perfect, but does its job.

**Comment by sallar** [comment]
@ph1p looks awesome! I’ll review tomorrow :) thanks

**Comment by ph1p** [comment]
Don't know why appveyor throws so often errors ... ¯\_(ツ)_/¯

**Comment by sallar** [comment]
@ph1p because its a bad system :(

File: package.json [hunk 1]
```diff
@@ -108,6 +127,7 @@
   },
   "homepage": "https://buttercup.pw",
   "dependencies": {
+    "assert": "^1.4.1",
```
Comment (sallar) [inline package.json]: Can you move this to `devDependencies` please?

Comment (ph1p) [reply package.json]: jep

**Comment by ph1p** [comment]
Jep. I've tested both cases.

**Pull request merged**